### PR TITLE
Add: Stock class

### PR DESCRIPTION
**Pull request by janebautina**
Add new change

### CODE REVIEW

File: Stock.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+public class Stock {
+    public int maxProfit(int[] prices) {
+        if(prices == null | prices.length == 0)
```
Comment (mbautin) [inline Stock.java]: Nit: add a space after `if`

Replace `|` with `||`

File: Stock.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+          return 0;
+        int maxP = Integer.MIN_VALUE;
+        int localMin = prices[0];
+        for (int i = 0; i < prices.length; i++) {
+            if(prices[i] >= localMin) {
```
Comment (mbautin) [inline Stock.java]: Add space after `if`

File: Stock.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+                if(prices[i] - localMin > maxP)
```
Comment (mbautin) [inline Stock.java]: Add space after `if`

File: Stock.java [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+                    maxP = prices[i] - localMin;
+            }
+            else{
```
Comment (mbautin) [inline Stock.java]: Add space before `{`

**Review by mbautin — CHANGES_REQUESTED** [review]

**Review by mbautin — APPROVED** [review]

File: Stock.java (hunk 1 above)
Comment (janebautina) [reply Stock.java]: Done.

**Pull request merged**